lib/src/model/game/game_repository.dart

```diff
@@ -51,6 +51,28 @@ class GameRepository {
     );
   }
 
+  Future<IList<LightArchivedGame>> getFullGames(
```

Comment:
This method should now replace the `getRecentGame` because both almost do the same thing.
So let's remove `getRecentGames`, and have a default `max` parameter at 20.